otto.go

```diff
@@ -775,3 +779,11 @@ func (o Object) MarshalJSON() ([]byte, error) {
 	}
 	return json.Marshal(goValue)
 }
+
+type Option func(*Otto)
+
+func WithStdoutWriter(w io.Writer) Option {
+	return func(o *Otto) {
+		o.runtime.stdoutWriter = w
```

Comment:
suggestion: lets document the type and change the naming so it easier for the user to understand the impact of using this option.
```suggestion
// WithConsoleOutput sets the target for console output.
// Default: Stdout.
func WithConsoleOutput(w io.Writer) Option {
	return func(o *Otto) {
		o.runtime.console = w
```